apps/desktop/desktop_native/core/src/password/unix.rs

```diff
@@ -1,106 +1,109 @@
 use anyhow::{anyhow, Result};
-use libsecret::{password_clear_sync, password_lookup_sync, password_store_sync, Schema};
+use oo7::dbus::{self};
 use std::collections::HashMap;
 
-pub fn get_password(service: &str, account: &str) -> Result<String> {
-    let res = password_lookup_sync(
-        Some(&get_schema()),
-        build_attributes(service, account),
-        gio::Cancellable::NONE,
-    )?;
+pub async fn get_password(service: &str, account: &str) -> Result<String> {
+    match get_password_new(service, account).await {
+        Ok(res) => Ok(res),
+        Err(_) => {
+            get_password_legacy(service, account).await
+        }
+    }
+}
 
+async fn get_password_new(service: &str, account: &str) -> Result<String> {
+    let keyring = oo7::Keyring::new().await?;
+    let attributes = HashMap::from([("service", service), ("account", account)]);
+    let results = keyring.search_items(&attributes).await?;
+    let res = results.get(0);
     match res {
-        Some(s) => Ok(String::from(s)),
-        None => Err(anyhow!("No password found")),
+        Some(res) => {
+            let secret = res.secret().await?;
+            Ok(String::from_utf8(secret.to_vec())?)
+        },
+        None => Err(anyhow!("no result"))
     }
 }
 
-pub fn get_password_keytar(service: &str, account: &str) -> Result<String> {
-    get_password(service, account)
+// forces to read via secret service; remvove after 2024.03
+async fn get_password_legacy(service: &str, account: &str) -> Result<String> {
+    println!("falling back to get legacy {} {}", service, account);
+    let svc = dbus::Service::new().await?;
+    let collection = match svc.default_collection().await {
+        Ok(c) => Ok(c),
+        Err(e) => Err(e),
+    }?;
```

Comment:
This match doesn't seem to be doing anything, I think you should be able to do this:
```suggestion
    let collection = svc.default_collection().await?;
```